### ISSUE TITLE
Prometheus Datasource problems

### ISSUE THREAD

**Issue by Xantorero** [metrico/qryn#392]
Hello! Today I have upgraded the setup with Qryn from version 2.4.4 to 3.0.15. I have encountered two issues while trying to get metric from Qryn as Prometheus Datasource in Grafana (version 10.2.1):

- On version 3.0.15 when I create some query I get:
![Error1](https://github.com/metrico/qryn/assets/27970408/e3c5dcbf-4fc5-4639-9c90-e87dd143e0ba)

- After the above error I tried to downgrade to 3.0.14. The ":b is not defined" error did not happen and I was able to query metrics successfuly. But the issue started to happen after some time. At first after selecting the time period on grafan to 5min there was suddenly "No Data" error. But when I increased the time period to 15min the data was shown properly, even metrics from last 5min were shown. Here it gets interesting - the more more time flown, the bigger time periods were suddenly unaviable with "No data". Now I am able to only query time period 12 hours:
![Error2](https://github.com/metrico/qryn/assets/27970408/ec835176-6924-4c09-9472-b79951ad9bd9)

When I change time period to anything less than 12 hours, no data:
![Error3](https://github.com/metrico/qryn/assets/27970408/9973da34-3a7a-430d-8821-d95db0d035df)

I have totaly no idea why this is happening, the data is properly saved in Clickhouse as it is shown when the time period is proper, I tried different 3.0.X versions - the same result. Downgrade to 2.4.4 did resolve this issue. 

Best Regards,
Adam

**Comment by akvlad** [comment]
Hello @Xantorero 
Tried to fix the issue in in v3.0.16 .
Please check if `b is not defined` is fixed.
Can't reproduce the "no data" for now. Are there any logs inside the qryn app when the data is absent? Maybe there's some internal exceptions raised.

**Comment by Xantorero** [comment]
Hello! 
I have checked the new version 3.0.16 and the `b is not defined` is gone, thank You very much!
As for the time period issue it resolved itself, I went to Grafana in the morning and everythin was working well, no changes from yesterday (it was still 3.0.14). So I will close this isseu, if something happens in the future I will create another topic with more details!

Thank You again for quick response as always!
